arxivdigest/frontend/views/general.py

```diff
@@ -236,6 +237,11 @@ def submitFeedback():
     flash('Successfully sent feedback.', 'success')
     return redirect('/')
 
+@mod.route('/termsandconditions/', methods=['GET'])
+def termsandconditions():
+    """Returns terms and conditions page."""
+    return render_template('termsAndConditions.html')
```

Comment:
termsAndConditions.html => terms_and_conditions.html
(please also rename passwordChange.html and registerSystem.html to all-lowercase and _-separated)